server.js

```diff
@@ -2,8 +2,10 @@
 
 //Require used in servers instead of import
 const express = require('express');
-const axios = require('axios')
+const axios = require('axios');
 const cors = require('cors');
+const { acceptsEncodings } = require('express/lib/request');
+const { response } = require('express');
```

Comment:
Good work on updates here! I am not sure what these are but I am pretty sure we don't need them/can remove 👍